README.md

```diff
@@ -25,8 +25,13 @@ What's new in 1.4.0
 * Hardware spline/bezier tesselation in OpenGL, D3D11 and Vulkan (...)
 * Post-processing shaders in D3D11
 * Prescale UV setting removed, now the default (improves perf) (#9176)
+* High DPI display fixes
+* New audio setting to improve compatibility with Bluetooth headsets
+* Various desktop gamepad compatibility fixes
 * Assorted minor compatibility fixes, code cleanup and performance improvements
 
+Support for Symbian, Maemo and Blackbarry has been removed.
```

Comment:
"Barry, is that how you get ants? Yes it is other Barry, yes it is."

I'll fix.